classes/Access.php

```diff
@@ -330,6 +348,18 @@ public function updateLgcAccess($idProfile, $idTab, $lgcAuth, $enabled)
             $whereClauses[] = ' `slug` LIKE "'.$slugLike.'"';
         }
 
+        if ($addFromParent == 1)
+        {
+           foreach (self::findSlugByIdParentTab($idTab) as $child)
+           {
+               $child = self::sluggifyTab($child);
+               foreach ((array) self::getAuthorizationFromLegacy($lgcAuth) as $auth) {
+                    $slugLike = Db::getInstance()->escape($child.$auth);
+                    $whereClauses[] = ' `slug` LIKE "'.$slugLike.'"';
+               }
+           }
+        }
+
```

Comment:
Sorry to not explained it sooner, but we follow the PSR rules.

So please update your code like this:

```php
if ($addFromParent == 1) {
    foreach (self::findSlugByIdParentTab($idTab) as $child) {
        $child = self::sluggifyTab($child);
        foreach ((array) self::getAuthorizationFromLegacy($lgcAuth) as $auth) {
            $slugLike = Db::getInstance()->escape($child.$auth);
            $whereClauses[] = ' `slug` LIKE "'.$slugLike.'"';
        }
    }
}
```

Thank you!